modules/ocl/cl_context.cpp

```diff
@@ -458,10 +479,15 @@ CLContext::import_dma_image (const cl_import_image_info_intel &import_info)
 {
     cl_mem mem_id = NULL;
     cl_int errcode = CL_SUCCESS;
-    if (!is_valid())
+    cl_platform_id platform_id = _device->get_platform_id ();
+    if (!is_valid() || !platform_id)
         return NULL;
 
-    mem_id = clCreateImageFromFdINTEL (_context_id, &import_info, &errcode);
+    OCLCREATEIMAGEFROMFDINTEL *oclCreateImageFromFdINTEL =
+        (OCLCREATEIMAGEFROMFDINTEL *) clGetExtensionFunctionAddressForPlatform (platform_id, "clCreateImageFromFdINTEL");
+    XCAM_ASSERT (oclCreateImageFromFdINTEL);
```

Comment:
All extension function name should be defined with Macros. e.g
        #define OCL_EXT_NAME_CREATE_IMAGE_FROM_FD_INTEL "clCreateImageFromFdINTEL"